serverless/serverless-kafka.adoc

```diff
@@ -36,6 +36,14 @@ spec:
 // Install Kafka
 include::modules/serverless-install-kafka-odc.adoc[leveloffset=+1]
 
+// Configure TLS and SASL for Kafka
+== Configuring authentication for Kafka
+
+In production, Kafka clusters are often secured using the TLS or SASL authentication methods. This section shows how to configure the Kafka channel to work against a protected Red Hat AMQ Streams (Kafka) cluster using TLS or SASL.
+
+include::modules/serverless-kafka-tls.adoc[leveloffset=+1]
+include::modules/serverless-kafka-sasl.adoc[leveloffset=+1]
```

Comment:
Both of those docs lack any verification steps, so that admin can make sure he configured it right, and KnativeKafka can work.

Reply:
Verification steps are optional for OpenShift docs in the docs guidelines, but yes it would be good to have these, thanks @cardil 🙂

Reply:
I've checked this a little. There is no knowing that the configurations is correct or not. `KnativeKafka` always reports ready, no matter if given parameters are correct or not. For example, if I mistype the secret name, `KnativeKafka` CR reports Ready, but `kafka-ch-dispacher` is showing errors in logs like:

```json
{
  "level": "error",
  "ts": "2021-03-22T21:59:34.082Z",
  "logger": "kafkachannel-controller",
  "caller": "utils/util.go:88",
  "msg": "Referenced Auth Secret not found",
  "knative.dev/pod": "kafka-ch-controller-69bdfc4f8d-stz57",
  "stacktrace":"knative.dev/eventing-kafka/pkg/channel/consolidated/utils.GetKafkaAuthData\n\t/opt/app-root/src/go/src/knative.dev/eventing/pkg/channel/consolidated/utils/util.go:88\nknative.dev/eventing-kafka/pkg/channel/consolidated/reconciler/controller.(*Reconciler).updateKafkaConfig\n\t/opt/app-root/src/go/src/knative.dev/eventing/pkg/channel/consolidated/reconciler/controller/kafkachannel.go:586\nknative.dev/eventing-kafka/pkg/channel/consolidated/reconciler/controller.NewController.func1\n\t/opt/app-root/src/go/src/knative.dev/eventing/pkg/channel/consolidated/reconciler/controller/controller.go:92\nknative.dev/pkg/configmap.(*ManualWatcher).OnChange\n\t/opt/app-root/src/go/src/knative.dev/eventing/vendor/knative.dev/pkg/configmap/manual_watcher.go:62\nknative.dev/pkg/configmap.(*InformedWatcher..."
}
```

Created a bug for this: [SRVKE-754](https://issues.redhat.com/browse/SRVKE-754)

Reply:
@cardil I'm assuming that at the moment we don't have a good verification procedure we could add? Please unresolve this if I misunderstood.

Reply:
Yes, at this point after editing the KnativeKafka values, users need to look at logs of `kafka-ch-dispacher` pod to see if no errors are there.

BTW. I can't unresolve thread.